### ISSUE TITLE
'make docs' results in "ERROR: tox config file not found"

### ISSUE THREAD

**Issue by ghost** [hyperledger/fabric#4132]
Following something as trivial as

https://hyperledger-fabric.readthedocs.io/en/latest/docs_guide.html#building-locally

## Root Cause

* Found via: https://stackoverflow.com/a/62413225/21521485

I use a docker instance for development (usually one ubuntu-docker-container per project). 

The relevant command

https://github.com/hyperledger/fabric/blob/42240f8b7fe767d3492984bb1988f0a348a6e0f1/Makefile#L342-L344

~uses~ runs on my setup as "docker-in-docker", which works fine, except the `-v` part, which mounts the volume of the HOST OS.

`/docs` is empty, thue the tox-container does not find the ini file

### Workaround

?

### Solution

?

**Comment by denyeart** [comment]
For Windows WSL2 is the recommended path, have you tried that? https://hyperledger-fabric.readthedocs.io/en/latest/dev-setup/devenv.html#developing-on-windows

Since most of the Fabric contributors use macOS, any enhancement PRs for the Windows specific documentation would be appreciated!

**Comment by ghost** [comment]
> For Windows WSL2 is the recommended path, have you tried that?

No, my experience says that hyper-v docker containers have less trouble than WSL2 (which I do not use at all)

But I can give it a try to see if I get the same error.
